### PR TITLE
[[ Merge 6.7.6 rc 1 ]]

### PR DESCRIPTION
**Pull request by livecodesebastien**
The only thing refactored is the new dontuseqt player property.

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 266c212

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 266c2129670358fb2fd16ecaa265decc1b89c280

File: engine/src/player-platform.cpp [hunk 1]
```diff
@@ -879,6 +879,9 @@ MCPlayer::MCPlayer()
     m_is_attached = false;
     m_should_attach = false;
     m_should_recreate = false;
+    
+    dontuseqt = false;
+    MCdontuseQT = false;
 }
```
Comment (livecodepanos) [inline engine/src/player-platform.cpp:885]: This should be dontuseqt = False and usingqt = False :)

Comment (livecodesebastien) [inline engine/src/player-platform.cpp:885]: Good catch.

Comment (livecodeali) [inline engine/src/player-platform.cpp:885]: Oops. Yes, thanks @runrevpanos.

**Pull request merged**